clifford/__init__.py

```diff
@@ -232,15 +238,21 @@ def _get_mult_function_runtime_sparse(k_list, l_list, m_list, mult_table_vals, n
     """
     @numba.njit
     def mv_mult(value, other_value):
-        output = np.zeros(n_dims)
+        # choose an appropriate output type
+        ret_dtype = (value[:0] * mult_table_vals[:0] * other_value[:0]).dtype
+
+        # this casting should be done at jit-time
+        mult_table_vals_t = mult_table_vals.astype(ret_dtype)
```

Comment:
Citation needed. Godbolt seems to show that at -O3, the equivalent C code is optimized out: 

https://godbolt.org/z/GsE2bB